src/rules/enforce-callback-memo.ts

```
@@ -31,6 +31,89 @@ export default createRule<[], MessageIds>({
+            }
+          });
+          return params;
+        }
+        current = current.parent;
+      }
+
+      return [];
+    }
+
+    function referencesParentScopeVariables(
+      functionNode: TSESTree.ArrowFunctionExpression | TSESTree.FunctionExpression,
```

ESLint

[error] Replace `·TSESTree.ArrowFunctionExpression` with `⏎········|·TSESTree.ArrowFunctionExpression⏎·······`

(prettier/prettier)